entc/load/schema.go

```diff
@@ -37,6 +37,7 @@ type Field struct {
 	Tag           string          `json:"tag,omitempty"`
 	Size          *int64          `json:"size,omitempty"`
 	Enums         []string        `json:"enums,omitempty"`
+	EnumDefault   string          `json:"enum_default,omitempty"`
```

Comment:
I don't think a new field is required here. Maybe we can just change the `Default` field to be type `interface{}`.

Let me know if you are fine with this, and I can help with the implementation part.

Reply:
As far as I understand right now the `load.Field.Default` is a bool and is used to know if that field has a default value set, it does not hold the actual value, right? That's why I came up with that `EnumDefault`, which is not the best solution, I agree, but I didn't want to mess up with the whole thing.